deployment/CP05MOAS-PG583_Deploy.csv

```diff
@@ -5,4 +5,11 @@ SI20170418,,TS20170727,,CP05MOAS-PG583-02-DOSTAM000,1,1,2017-04-19T05:30:47,2017
 SI20170418,,TS20170727,,CP05MOAS-PG583-03-FLORTM000,1,1,2017-04-19T05:30:47,2017-07-27T09:10:00,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-FLORTM-04093,40.37043,-70.95436,,200,200,CNPM
 SI20170418,,TS20170727,,CP05MOAS-PG583-04-FLORTO000,1,1,2017-04-19T05:30:47,2017-07-27T09:10:00,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-FLORTO-01349,40.37043,-70.95436,,200,200,CNPM
 SI20170418,,TS20170727,,CP05MOAS-PG583-05-NUTNRM000,1,1,2017-04-19T05:30:47,2017-07-27T09:10:00,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-NUTNRM-00709,40.37043,-70.95436,,200,200,CNPM
-SI20170418,,TS20170727,,CP05MOAS-PG583-06-PARADM000,1,1,2017-04-19T05:30:47,2017-07-27T09:10:00,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-PARADM-50218,40.37043,-70.95436,,200,200,CNPM
\ No newline at end of file
+SI20170418,,TS20170727,,CP05MOAS-PG583-06-PARADM000,1,1,2017-04-19T05:30:47,2017-07-27T09:10:00,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-PARADM-50218,40.37043,-70.95436,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-00-ENG000000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGCON-GLDRGP-00583,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-01-CTDGVM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-CTDGVM-09307,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-02-DOSTAM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-DOSTAM-00539,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-03-FLORTM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-FLORTM-04093,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-04-FLORTO000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-FLORTO-01348,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-05-NUTNRM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-NUTNRM-00709,40.08518,-70.36618,,200,200,CNPM
+TS20200613,,,,CP05MOAS-PG583-06-PARADM000,2,1,2020-06-13T20:10:46,,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-PARADM-50218,40.08518,-70.36618,,200,200,CNPM
```

Comment:
looks good

Reply:
CGVEH-GLDRGP-00583  should be GLDRCP  (this needs to be added to the node_bulk_load-AssetRecord. 
CGCON-GLDRGP-00583 should be GLDRCP  (this already exists in the eng_bulk_load-AssetRecord.